### PR TITLE
Prime Tricks

### PR DESCRIPTION
**Pull request by toasterparty**

### CODE REVIEW

File: randovania/games/prime1/json_data/Chozo Ruins.txt [hunk 1]
```diff
@@ -1257,15 +1261,22 @@ Extra - asset_id: 1227669322
...
               All of the following:
                   Space Jump Boots
                   Any of the following:
                       Gravity Suit
                       Slope Jump (Beginner) and Gravityless Underwater Movement (Beginner)
+              All of the following:
+                  Boost Ball and Gravity Suit and Morph Ball and Wall Boost (Beginner)
+                  Any of the following:
+                      Movement (Intermediate) and Poisoned Water Damage ≥ 65
+                      Movement (Expert) and Poisoned Water Damage ≥ 20
```
Comment (dyceron) [inline randovania/games/prime1/json_data/Chozo Ruins.txt:1279]: If I'm reading this right, there is an "All of the following" that groups everything here together, which I feel is a mistake.

**Review by dyceron — CHANGES_REQUESTED** [review]
Changes highlighted need clarification

File: randovania/games/prime1/json_data/Chozo Ruins.txt (hunk 1 above)
Comment (toasterparty) [reply randovania/games/prime1/json_data/Chozo Ruins.txt:1279]: image
This is the 4th of 4 options to get to the item. The first all of the following is just a way of using different damage requirements for walking to the base of the vertical obstacle (you can walk faster underwater w/ grav).

Comment (dyceron) [reply randovania/games/prime1/json_data/Chozo Ruins.txt:1279]: Oh that makes sense. Yeah this is fine then.

**Pull request merged**